app/models/financial_year.rb

```diff
@@ -0,0 +1,38 @@
+class FinancialYear
```

Comment:
Looks like there might be some behaviour shared between this and `FinancialPeriod` (defined in `lib/financial_period.rb`).

Reply:
Ah, good call. I've done a bit of refactoring to get rid of that module altogether, and introduce a new FinancialQuarter class. Would appreciate a look before merging if that's cool?